types/minetest/minetest.d.tl

```diff
@@ -1,86 +1,107 @@
--- TODO: math.pi
-
-function loadstring(x: string): any, string
+function loadstring(x: string): any, string -- TODO: add to tl
 end
 
-function setfenv(x: any, y: any) -- TODO
+function setfenv(x: any, y: any) -- TODO: add to tl
```

Comment:
A quick heads-up: I'd rather not adding Lua 5.1-only functions to the Teal standard library definitions — that reminds me I might need to double-check the `math` library additions!